LEGO1/mxdsaction.h

```diff
@@ -49,6 +49,9 @@ class MxDSAction : public MxDSObject
 
   inline MxU32 GetFlags() { return this->m_flags; } 
   inline void SetFlags(MxU32 m_flags) { this->m_flags = m_flags; }
+  inline char *GetUnkData() { return m_unkData; }
```

Comment:
I think we can rename the variables, functions etc. to `m_extraData`, `GetExtraData` etc. - it seems clear to me now this is just the "extra" data component of the SI files, also referenced in @itsmattkc newest YouTube video (which isn't public yet but accessible to patterns)

Reply:
(Spoiler alert!) I saw that name "extra" in the SIEdit repo earlier too. I changed it all throughout MxDSAction.